script/controller/tasklist-front.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @author Bogdan Kovalev
+ */
+
+angular.module('tasklist-front', ['tasklist-back', 'utils'])
+
+    .controller('widgetController', ['$scope', '$state', 'Tasks', 'Utils', function ($scope, $state, Tasks, Utils) {
+
+        $scope.description = "";
+        $scope.assignee = "";
+        $scope.onlyAssigned = $state.is('tasks.assigned');
+
+        $scope.items = Tasks.getItems();
+
+        $scope.items.forEach(function (item) {
+            item.hovered = false;
```

Comment:
We discussed that on a lecture: model should not be aware of ui-related stuff.
selected is OK, but hovered and focused is too ui-specific